packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -369,11 +369,13 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
                             return errorBuilder(context, snapshot.error!);
                           }
                         }
+
                         if (snapshot.connectionState == ConnectionState.done) {
                           return Stack(children: stackedWidgets);
                         }
+
                         return widget.loadingBuilder?.call(context) ??
-                            Container();
+                            const SizedBox.shrink();
```

Comment:
From the container api docs: 
> If the widget has no child, no height, no width, no [constraints](https://api.flutter.dev/flutter/widgets/Container/constraints.html), and no [alignment](https://api.flutter.dev/flutter/widgets/Container/alignment.html), but the parent provides bounded constraints, then [Container](https://api.flutter.dev/flutter/widgets/Container-class.html) expands to fit the constraints provided by the parent.

To preserve behavior, we should expand. 

```suggestion
                            const SizedBox.expand();
```